### DISCUSSION TITLE
Is Swup abandoned?

### DISCUSSION THREAD

**Discussion by mevanloon** [swup/swup#424] [Help] [answered]
I'm still using Swup on a bunch of sites because of its speed, but I noticed that there haven't been any commits in a bit more than a year. There are pull requests by dependabot not being addressed either.

**Comment by gmrchk** [comment]
Hi, @mevanloon! 👋  Good question. Glad you asked!

As you and a few others noticed, I've been very passive when it comes to Swup. The truth is I have not been using the library myself in ages. The area of my work simply shifted to different places and Swup is no longer a tool I require for my daily work. 

Unfortunately, that means two things. 
1. I can no longer provide proper support to people because I can't go back and forth to issues that are not relevant to me. I have to research the problem every time, and that's not something feasible timewise. Luckily, the community around Swup has been taking care of that for the most part. 
2. It is very hard for me to imagine a correct future path for a tool I don't use myself. 

## Now to your question - is Swup abandoned?
No, I'm aware of the problem and looking for a solution. More precisely, I'm in search of maintainers that can provide what I no longer can. I'm hoping for these people to take over the project, including shaping the future versions of Swup and bringing new ideas to the table. 
I am not going anywhere and will be always around, but I can no longer actively lead a project without even using it. Ideally, I would move into a more organizational role, making sure the people behind Swup can work and improve it effectively. 

If anyone would like to sign up for the job or would like to propose a candidate, please [let me know](mailto:swup@gmrchk.com). 
The ideal candidate should be using Swup actively in their projects - the more, the better. This is not a one-man job and I'm counting on more people being involved, so don't worry about being overwhelmed. I'm in contact with a few people already too, so expect progress soon.

**Reply by gmrchk** [reply]
@AugustMiller I've been having trouble contacting you by email. As a super active user, I would be happy for you to join as well. If you're interested, please, [do let me know](mailto:gmarcuk@gmail.com).

**Comment by gmrchk** [comment, marked as answer]
With the v3 and v4 released since this question was posted, and awesome active development mainly from @daun and @hirasso, I think it's fair to say Swup is not abandoned.
